test/com/comeet/utilities/ExchangeUtility.java

```diff
@@ -0,0 +1,28 @@
+package com.comeet.utilities;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import microsoft.exchange.webservices.data.core.ExchangeService;
+import microsoft.exchange.webservices.data.core.enumeration.misc.ExchangeVersion;
+import microsoft.exchange.webservices.data.credential.ExchangeCredentials;
+import microsoft.exchange.webservices.data.credential.WebCredentials;
+
+public class ExchangeUtility {
+    
+    /**
+     * Creates an exchange service to do tests.
+     * @throws URISyntaxException if service fails
+     */
+    public static ExchangeService getExchangeService(String username, String password) throws URISyntaxException {
```

Comment:
To make the unit tests fast and isolated, we should mock out the ExchangeService passed back during tests.